ArjunaCore/arjuna/pom.xml

```diff
@@ -461,7 +437,6 @@
               <excludes>
               </excludes>
             </configuration>
-
```

Comment:
This is just formatting change to remove the line

Reply:
I preferred to restore the line. Thanks Tom